oci-artificial-intelligence/ai-speech/customizations/customizations.md

```diff
@@ -103,11 +103,162 @@ To view your newly created customization, click on your customization's name fro
     From either the customization details page or list customizations page, click <strong>delete</strong> to delete your customization
         ![delete customization](./images/delete-customization.png " ")
 
+# Task 4: Using the customizations API
+
+The Postman REST Collection for Customizations setup can be downloaded from [OCI Customization REST Collection](./files/Customization_API_Collection.postman_collection.json). The POSTMAN instructions for the setups are [the same as Speech API](../ai-speech/oci-speech-rest/oci-speech-rest.md)
+
+OCI Speech Service EndPoints for all the services:
+
+*Note:* The list of region identifiers for each region can be found [here](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm), update the endpoints with appropriate region identifiers.
+
+1. <u>Create Customization Job</u>
+    ![Create customization job](./images/customization-rest-post.png)
```

Comment:
I'd say we go without the postman screenshots here since we already have all the info written on the page